cli/Cargo.toml

```diff
@@ -1,7 +1,18 @@
 [workspace]
 members = ["crates/*"]
 
+# TODO: Probably only do these on CI if they work
+
+[profile.dev]
```

Comment:
`CARGO_PROFILE_DEV_DEBUG=0` in the CI instead? While I'm rarely debugging, I would still be surprised to not see any debug info locally.